### PR TITLE
Smarty component documentation

### PR DESCRIPTION
**Pull request by z0mbieprocess**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#21491
## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/smarty.markdown [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+layout: page
+title: "Salda Smarty 2X/3X/4X P/V Ventilation"
+description: "Instructions on how to integrate Salda Smarty 2X/3X/4X P/V ventilation systems into Home Assistant."
+date: 2019-02-25 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: salda.png
+ha_category: Hub
```
Comment (klaasnicolaas) [inline source/_components/smarty.markdown]:
```suggestion
ha_category:
  - Hub
  - Fan
  - Sensor
```

File: source/_components/smarty.markdown [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ha_release: "0.89"
+---
+
+The `smarty` component lets you control Salda [Smarty](http://www.salda.lt/en/products/category/compact-counter-flow-units) ventilation units from Home Assistant. You need a [MB-GATEWAY](http://www.salda.lt/en/products/item/5637227077) or something similar to connect to your local network.
+
```
Comment (klaasnicolaas) [inline source/_components/smarty.markdown:19]:
```suggestion
There is currently support for the following device types within Home Assistant:

- Fan
- Sensor
```

File: source/_components/smarty.markdown [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+- Filter Change Timer
+
+## {% linkable_title Configuration %}
+
+To set it up, add the following information to your `configuration.yaml` file:
+
+```yaml
+smarty:
+  host: 192.168.0.10
+  name: Smarty 3X
```
Comment (klaasnicolaas) [inline source/_components/smarty.markdown]: `name` is optional and does not belong in the first example config

**Review by klaasnicolaas — DISMISSED** [review]

**Comment by klaasnicolaas** [comment]
After these changes you can delete the `fan` and `sensor` page

**Review by frenck — APPROVED** [review]
Thanks, @z0mbieprocess! 👍

**Pull request merged**